source/runtime.hpp

```diff
@@ -31,6 +31,29 @@ namespace reshade
 	class runtime abstract
 	{
 	public:
+		enum depth_buffer_retrieval_mode
+		{
+			POST_PROCESS = 0,
```

Comment:
Also, naming convention: These should be snake_case.

Reply:
fixed